tapir/wirgarten/models.py

```diff
@@ -0,0 +1,73 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+from django.utils.translation import gettext_lazy as _
+from localflavor.generic.models import IBANField, BICField
+
+from tapir.accounts.models import TapirUser
+
+
+class PickupLocation(models.Model):
+    name = models.TextField(_("Name"), max_length=150)
+    coords_lon = models.DecimalField(
+        _("Coordinates Longitude"), decimal_places=10, max_digits=12
+    )
+    coords_lat = models.DecimalField(
+        _("Coordinates Latitude"), decimal_places=10, max_digits=12
+    )
+    street = models.CharField(_("Street and house number"), max_length=150)
+    street_2 = models.CharField(_("Extra address line"), max_length=150, blank=True)
+    postcode = models.CharField(_("Postcode"), max_length=32)
+    city = models.CharField(_("City"), max_length=50)
+    info = models.CharField(_("Additional info like opening times"), max_length=150)
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(
+                fields=["name", "coords_lat", "coords_lon"],
+                name="unique_location",
+            ),
+        ]
+
+
+class Member(TapirUser, models.Model):
```

Comment:
Would it make sense to have a WirgartenUserData model that would have a 1:1 relation with TapirUser, similar to [ShiftUserData](https://github.com/FoodCoopX/wirgarten-tapir/blob/9db7461bbc5c7959e17c90af4d8fdb0b7d28a445/tapir/shifts/models.py#L850)?

Reply:
This is already a seperate table with 1:1 relation to TapirUser.